src/components/CartsContainer/CartsContainer.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+
+import { confusedIMG } from '../../assets/resources/imagesURL';
+import { deleteCurrentPhoneCart, clearCurrentCart } from '../../store/actions/inCartAction';
+import './CartsContainer.scss';
+
+const CartsContainer = (props) => {
+
+   const { cart, totalAmount, deleteCurrentPhoneCart, clearCurrentCart } = props;
+   const testIMG = 'https://s.abcnews.com/images/Technology/samsung-phone-gty-jpo-190423_hpMain_16x9_992.jpg';
+
+   const clearCartHandler = () => {
+      clearCurrentCart([]);
```

Comment:
Just have a action without payload and replace cart with empty array in reducer
```suggestion
      clearCurrentCart();
```